### PR TITLE
Refactor doc test and justfile

### PR DESCRIPTION
**Pull request by yihuaf**
- Separated doc test into a separated justfile recipe
- Refactor the name of the test related to recipe to follow the same pattern of test-*.

### CODE REVIEW

File: justfile [hunk 1]
```diff
@@ -90,7 +96,7 @@ bin-kind:
 # Clean
 
 # Clean kind test env
-clean-test-kind:
+test-kind-clean:
```
Comment (YJDoc2) [inline justfile:99]: I think this should be `clean-test-kind` as this is not a test, but a cleanup script instead.

File: justfile [hunk 2]
```diff
@@ -28,16 +28,22 @@ rust-oci-tests-bin:
...
-# run cargo unittests
-unittest:
-    cd ./crates
-    LD_LIBRARY_PATH=${HOME}/.wasmedge/lib cargo test --all --all-targets --all-features
+# run basic tests
+test-basic: test-unit test-doc
+
+# run cargo unit tests
+test-unit:
+    cargo test --lib --bins --all --all-targets --all-features
```
Comment (YJDoc2) [inline justfile:42]: Do we not need the `LD_LIBRARY_PATH` ?

**Review by YJDoc2 — COMMENTED** [review]
overall fine, some nits

**Comment by YJDoc2** [comment]
@yihuaf , if you're busy with something else, just reply to the above comments, and I can push a commit that makes changes accordingly and merge :+1:

**Comment by yihuaf** [comment]
Please go ahead. Thanks!

**Comment by YJDoc2** [comment]
> Please go ahead. Thanks!

Sure, no worries!

**Review by YJDoc2 — APPROVED** [review]

**Pull request merged**